locales/en/messages.json

```
@@ -4858,6 +4858,13 @@
     "osdDescElementVtxChannel": {
         "message": "Current VTX channel and power"
     },
+    "osdTextElementVTXchannelVariant-power": {
+        "message": "VTX power",
+        "description": "One of the elements of the OSD"
```

To be in line with other variants:
```suggestion
        "description": "One of the variants of the altitude element of the OSD"        
```